### ISSUE TITLE
cannot subscript a string LGroup key

### ISSUE THREAD

**Issue by gdementen** [liam2/larray#437]
Ironically, it works for PGroup:

```python
>>> a = Axis('a=a0,a1')
>>> a.i[0][1:]
'0'
>>> a['a0'][1:]
TypeError: cannot take a subset of a0 because it has a '<class 'str'>' key
>>> # one can workaround this by using str() on the group, but that should not be necessary
>>> str(a['a0'])[1:]
'0'
```

**Comment by gdementen** [comment]
I think this is simply the test at: 
https://github.com/liam2/larray/blob/master/larray/core/group.py#L974

which is bogus. I guess it only needs to be changed to:
```python
elif np.isscalar(orig_key):
```

**Comment by alixdamman** [comment]
closed by PR #561. Wrong commit message.
